### ISSUE TITLE
BanManager Support

### ISSUE THREAD

**Issue by ahdg6** [samerton/Nameless-Infractions#11]
Add support for [BanManager](https://banmanagement.com/features)
It's Open Source and has own WebUI which means it will be easier to integrate

**Comment by austin4195** [comment]
That would be nice as it is one of the few options for Sponge servers that don’t run a proxy like Bungee

**Comment by Blocksnmore** [comment]
advancedban is a thing you know?

**Comment by austin4195** [comment]
Yes, but there isnt a Sponge version of AdvancedBan to my knowledge.

**Comment by ahdg6** [comment]
Banmanager has existed for a long time since 2012. It comes from bukkit and is not release on the spigot page, but it is more stable and practical in some ways

**Comment by austin4195** [comment]
Plus if you search "ban" on Sponge's Ore, there are only 2 plugins that come up with SQL support and BanManager has more downloads and a more stable codebase because it has been around for so long.
